aksk/pkg/iam/driver/keystone/keystone.go

```diff
@@ -220,3 +247,36 @@ func (iam *KeystoneIam) DownloadAkSk(ctx context.Context, in *pb.GetAkSkRequest)
 	json.Unmarshal(bodyBytes, &akskListout)
 	return akskListout, nil
 }
+
+func (iam *KeystoneIam) ValidateId(id string, token string, uri string) (bool, error) {
+
+	// Validate userId , tenantId are legitimate
+	log.Info("Validating ID ", id)
```

Comment:
for better debugging, will it be good to enclose the 'id' under some braces or delimiter. ex, id contains some spaces, then debugging may be easier

Reply:
done